### DISCUSSION TITLE
Better control by roles

### DISCUSSION THREAD

**Discussion by Kvarkas** [directus/directus#23129] [Feature Requests]
### Summary

Using latest Directus, I want to control what users could see after login, currently I can't restrict by role items in menu (but I could remove them globally as work around) however then user click on his icon - he able to see all roles in a system, that should not be and I want to believe its should be done by role? 

### Basic Example

![image](https://github.com/user-attachments/assets/0b195a79-4c7f-4f26-b568-1324351e3511)

### Motivation

Have full control on what user see in a directus after login

### Detailed Design

expecting more granular control for a roles

### Must Have:
- role control over menu items (then we create item, should be an option to choose whats roles are able to see that)
- similar as above, visibility control for other objects what are not belong to user with that role (show/hide)

### Drawbacks

have no idea :D 

### Alternatives

not sure, but that seems to be oblivious for me to have :)

### Adoption Strategy

I expect no distrubtion for already running projects as this will be new functionality and default settings could be allow to all roles

**Comment by rijkvanzanten** [comment]
This is based on permissions, you can reduce the user's permissions to see less or no other roles 👍🏻
